### PR TITLE
ON-6461 # Added memorySize property

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: docs/api/memory-constraints.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+We allow for increasing the Memory Size of your api through the `server.memorySize` property in the `.blinkmrc.json` file;
+
+#### .blinkmrc.json
+
+The following configuration will allow 2GB for your api functions:
+
+```json
+{
+  "server": {
+    "memorySize": 2048
```
Comment (mymattcarroll) [inline docs/api/memory-constraints.md:18]: I expect aws only allows a certain number of values here, we should document what they are.

File: lib/api/deploy.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ const EXT = 'zip'
 const HANDLER = 'handler'
 const pMkdir = util.promisify(temp.mkdir)
 
-const getProjectPath = (target) => path.join(target, 'project')
+const getProjectPath = target => path.join(target, 'project')
```
Comment (mymattcarroll) [inline lib/api/deploy.js:41]: You have prettier problems, make sure you have done `npm i` and restarted VS Code.

Comment (Zaxist) [reply lib/api/deploy.js:41]: Isn't this how it should be formatting it?

File: docs/api/memory-constraints.md (hunk 1 above)
Comment (Zaxist) [reply docs/api/memory-constraints.md:18]: Yeh, multiples of 64, Ill put it in the doc

File: lib/api/deploy.js (hunk 2 above)
Comment (mymattcarroll) [reply lib/api/deploy.js:41]: nope

File: docs/api/memory-constraints.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# OneBlink API CLI
+
+## Memory Constraints
+
+By default Memory for your api is limited to 1024 MB (1 GB).
+
+### Increasing Memory Size
+
+We allow for increasing the Memory Size of your api through the `server.memorySize` property in the `.blinkmrc.json` file. Only multiples of `64` are allowed;
```
Comment (mymattcarroll) [inline docs/api/memory-constraints.md:9]: Minimum and maximum?

**Pull request merged**